### PR TITLE
Financial institution search

### PR DESCRIPTION
**Pull request by bkmoovio**
#5

### CODE REVIEW

File: ACHDictionary.go [hunk 1]
```diff
@@ -15,19 +17,25 @@ import (
...
 	// errors holds each error encountered when attempting to parse the file
 	errors base.ErrorList
 }
 
 // NewACHDictionary creates a ACHDictionary
 func NewACHDictionary(r io.Reader) *ACHDictionary {
 	return &ACHDictionary{
-		IndexACHParticipant: map[string]*ACHParticipant{},
-		scanner:             bufio.NewScanner(r),
+		IndexACHRoutingNumber: map[string]*ACHParticipant{},
```
Comment (adamdecaf) [inline ACHDictionary.go]:
```suggestion
		IndexACHRoutingNumber: make(map[string]*ACHParticipant),
```

File: ACHDictionary.go [hunk 2]
```diff
@@ -15,19 +17,25 @@ import (
...
+		IndexACHCustomerName:  map[string][]*ACHParticipant{},
```
Comment (adamdecaf) [inline ACHDictionary.go]:
```suggestion
		IndexACHCustomerName:  make(map[string][]*ACHParticipant),
```

File: WIREDictionary.go [hunk 3]
```diff
@@ -15,19 +17,25 @@ import (
...
 	// errors holds each error encountered when attempting to parse the file
 	errors base.ErrorList
 }
 
 // NewWIREDictionary creates a WIREDictionary
 func NewWIREDictionary(r io.Reader) *WIREDictionary {
 	return &WIREDictionary{
-		IndexWIREParticipant: map[string]*WIREParticipant{},
-		scanner:              bufio.NewScanner(r),
+		IndexWIRERoutingNumber: map[string]*WIREParticipant{},
```
Comment (adamdecaf) [inline WIREDictionary.go]:
```suggestion
		IndexWIRERoutingNumber: make(map[string]*WIREParticipant),
```

File: WIREDictionary.go [hunk 4]
```diff
@@ -15,19 +17,25 @@ import (
...
+		IndexWIRECustomerName:  map[string][]*WIREParticipant{},
```
Comment (adamdecaf) [inline WIREDictionary.go]:
```suggestion
		IndexWIRECustomerName:  make(map[string][]*WIREParticipant),
```

**Comment by bkmoovio** [comment]
Closes #5

**Pull request closed without merging**